provider/core/Cargo.toml

```diff
@@ -36,7 +36,7 @@ icu_locid = { version = "0.2", path = "../../components/locid" }
 tinystr = "0.4.5"
 writeable = { version = "0.2", path = "../../utils/writeable" }
 thiserror = "1.0"
-yoke = { version = "0.2", path = "../../utils/yoke", features = ["serde"] }
+yoke = { version = "0.2", path = "../../utils/yoke", features = ["serde", "derive"] }
```

Comment:
Discussion: should we make the `, "derive"` be an optional feature, so we don't pull in `syn`?  If every data struct effectively requires the custom derive, though, maybe this is unavoidable.

Reply:
yeah everything needs the derive anyway. I guess utility crates could avoid it but I don't see much of a benefit